src/ui/widgets/video_capture.py

```
@@ -6,30 +6,53 @@
+        # Progress overlay at bottom center
+        self.overlay_progress_label = ctk.CTkLabel(
+            self, text="",
+            font=ctk.CTkFont(family="Poppins", size=16, weight="bold"),
+            fg_color="#222831", text_color="white", corner_radius=12, padx=12, pady=6
+        )
+        self.overlay_progress_label.place(relx=0.5, rely=0.92, anchor="center")
         
         self.cap = None
         self.view_model = view_model
         self.last_frame = None
         self.faces_with_status = [] # Initialize faces_with_status
```

Flake8 (7.2.0)

[error] at least two spaces before inline comment

(E261)